### PR TITLE
it's atlive!

### PR DESCRIPTION
**Pull request by mattkrick**

### CODE REVIEW

**Comment by jordanh** [comment]
Reviewing this now!

**Comment by jordanh** [comment]
My testing checklist:

- [ ] Welcome wizard -> new team -> new invitation generation
- [ ] Invitation acceptance
- [ ] Me dashboard -> navigate to team dashboard
- [ ] Team dashboard new project creation, status assignment -> back to Me dashboard
- [ ] Team dashboard multiplayer editing
- [ ] Start meeting, join second player
- [ ] Facilitate meeting until end

**Comment by jordanh** [comment]
Ran into something, steps:

1. Log in as `jordan@prbl.co`
2. Navigate to `/me`
3. Navigate to Parabol team dashboard

You get ducks.

Console error:

image

It happens a few time. The stack traces vary, so you should repro yourself.

If you reload the team route without navigating from `react-router`, it seems to work fine.

**Comment by jordanh** [comment]
Just made it through my suite of tests. I only found the one issue!

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "babel-runtime": "6.11.6",
     "bcrypt": "^0.8.7",
     "body-parser": "1.15.2",
-    "cashay": "^0.19.0",
+    "cashay": "^0.20.0",
```
Comment (jordanh) [inline package.json]: Probably be good to bump the Action app version too.

File: src/universal/modules/meeting/containers/Meeting/MeetingContainer.js [hunk 2]
```diff
@@ -33,21 +25,66 @@ import {
...
+        isConnected: teamMember.presence.length > 0,
+        isFacilitator: team.activeFacilitator === teamMember.id,
+        isSelf: teamMember.id.startsWith(userId)
+      };
+    }
+  }
+  return resolveMeetingMembers.cache;
+};
+
+const meetingContainerQuery = `
```
Comment (jordanh) [inline src/universal/modules/meeting/containers/Meeting/MeetingContainer.js]: This query is AMAZING.

File: src/universal/modules/teamDashboard/containers/AgendaListAndInput/AgendaListAndInputContainer.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import React, {PropTypes} from 'react';
+import {cashay} from 'cashay';
+import {connect} from 'react-redux';
+import AgendaListAndInput from 'universal/modules/teamDashboard/components/AgendaListAndInput/AgendaListAndInput';
+
+const agendaSubQuery = `
```
Comment (jordanh) [inline src/universal/modules/teamDashboard/containers/AgendaListAndInput/AgendaListAndInputContainer.js]: Again. This is AMAZING.

**Comment by mattkrick** [comment]
I can't seem to reproduce! Did you clear localStorage before testing?

**Comment by mattkrick** [comment]
🎵 find it fix it push it ship it 🎵 
bug killed!

**Comment by jordanh** [comment]
🎵  @directives make us stronger 🎵

**Comment by ackernaut** [comment]
👏

**Pull request merged**